config/default.js

```diff
@@ -139,15 +154,17 @@ module.exports = {
         // For backwards-compatibility, allow images to be served from the old
         // dedicated domain for statics and user uploaded content:
         'https://addons.cdn.mozilla.net/',
+        ga4AnalyticsHost,
+        ga4TagManagerHost,
       ],
       manifestSrc: ["'none'"],
       mediaSrc: ["'none'"],
       objectSrc: ["'none'"],
       // Script is limited to the static path
       scriptSrc: [
         `${baseUrlProd}${staticPath}`,
-        `${analyticsHost}/analytics.js`,
-        `${ga4Host}/gtag/js`,
+        ga4AnalyticsHost,  // https://www.google-analytics.com/analytics.js
+        ga4TagManagerHost, // https://www.googletagmanager.com/gtag/js
```

Comment:
I can see why these comments could be helpful... but they imply that, for example, the value of `ga4AnalyticsHost` is literally `https://www.google-analytics.com/analytics.js` when it's considerably looser that than - it's _any_ script under that domain or any subdomain.  

I lean towards removing them and allowing curious readers to read the comments in shared.js - otherwise, imo, they need to be clearer that `analytics.js` is the expected file, but just one of the possible allowed under the CSP.